test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Rename this file to integration-test.sh, otherwise it looks like it might just run unit tests